etl/steps/data/garden/artificial_intelligence/2025-11-11/nvidia_revenue.py

```diff
@@ -0,0 +1,51 @@
+"""Garden step for NVIDIA quarterly revenue data."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
```

Comment:
Avoid using `dest_dir`

```suggestion
def run() -> None:
```